pkg/cli/create/identity.go

```diff
@@ -58,7 +60,7 @@ func NewCmdCreateIdentity(f genericclioptions.RESTClientGetter, streams generici
 		Example: identityExample,
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(cmd, f, args))
-			cmdutil.CheckErr(o.Run())
+			ocmdhelpers.CheckGroupResourceNotRegisteredErr(o.Run(), o.DiscoveryClient, "Identity creation", ocmdhelpers.CustomOIDCReason)
```

Comment:
make the helper specifically for `"Identity creation", ocmdhelpers.CustomOIDCReason`.  We can create another custom one in a few years when we need it.

Reply:
Had this for `oc adm` user-related commands, `oc create useridentitymapping`, etc - which I would add once we liked the shape.